app/templates/project-name.Gulp/tasks/watch.js

```diff
@@ -1,38 +1,36 @@
+import gulp from 'gulp';
 import browserSync from 'browser-sync';
 import cfonts from 'cfonts';
-import gulp from 'gulp';
 import config from '../config';
 
 function watch() {
     cfonts.say(config.pkg.name, {
-        font: 'simple',
-        letterSpacing: 0,
-        space: false,
-        maxLength: '20'
+        'font': 'simple',
+        'letterSpacing': 0,
+        'space': false,
+        'maxLength': '20'
```

Comment:
Does cfonts somehow handles quoted props differently, like `'font': 'simple'` over `font: 'simple'`? Because I don't think it should and usually I only use quoted props when you have something like `'weird-thing': true`. :smile:

Reply:
It doesn't.. I think I pretty much always try to keep it safe.. but we can use it without the quotes for sure 😄